android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/CarJavaInterfaceTest.java

```diff
@@ -0,0 +1,23 @@
+package com.mapbox.maps.testapp.auto.car;
+
+import static com.mapbox.maps.extension.androidauto.MapboxCarUtilsKt.initMapSurface;
+
+import androidx.car.app.Session;
+
+import com.mapbox.maps.MapInitOptions;
+import com.mapbox.maps.extension.androidauto.MapSurfaceReadyCallback;
+import com.mapbox.maps.extension.androidauto.OnMapScaleListener;
+import com.mapbox.maps.extension.androidauto.OnMapScrollListener;
+
+public class CarJavaInterfaceTest {
```

Comment:
I'm not sure that it's a test. More like validator or checker

Reply:
Yes, just verify the Java API

Reply:
so maybe rename it?

Reply:
Renamed to xxxChecker